frontend/language/src/nb.json

```
@@ -1,202 +1,201 @@
+  "app_create_release.no_changes_on_current_release": "Det er ingen endringer på nåværende lansering",
   "app_create_release.ok": "Ok",
-  "app_create_release.release_creating": "Bygger en ny versjon av applikasjonen, startet av",
+  "app_create_release.release_creating": "Bygger en ny versjon av appen, startet av",
   "app_create_release.release_description": "Beskriv innholdet i versjonen",
-  "app_create_release.release_version_number": "Versjonsnummer",
-  "app_create_release.release_version_number_already_exists": "Et versjonnummer med det navnet finnes allerede.",
-  "app_create_release.release_version_number_validation": "Må være en unik kombinasjon av tall, små bokstaver (a-z) og spesialtegnene \".\" og \"-\". Maks 128 tegn.",
+  "app_create_release.release_versionnumber": "Versjonsnummer",
+  "app_create_release.release_versionnumber_already_exists": "Det finnes allerede et versjonnummer med dette navnet.",
+  "app_create_release.release_versionnumber_validation": "Må være en unik kombinasjon av tall, små bokstaver (a-z) og spesialtegnene \".\" og \"-\". Maks 128 tegn.",
   "app_create_release.status_popover": "Status for neste bygg",
   "app_create_release.still_building_release": "Bygger fortsatt versjon {{version}}",
-  "app_create_release_errors.check_status_on_build_error": "Oi! Vi opplever en teknisk feil og får derfor ikke oppdatert status på bygget. Vi forsøker å sjekke status på nytt...",
-  "app_create_release_errors.fetch_release_failed": "Huff da, vi opplever en teknisk feil som ikke gjør det mulig for deg å bygge versjoner eller deploye en applikasjon til miljøene akkurat nå. Prøv igjen senere. Dersom problemet vedvarer, kontakt <0 href=\"mailto:tjenesteeier@altinn.no\">Altinn servicedesk</0>.",
+  "app_create_release_errors.check_status_on_build_error": "Det har skjedd en teknisk feil, så vi får ikke oppdatert statusen på bygget ditt. Vi prøver å sjekke statusen på nytt …",
+  "app_create_release_errors.fetch_release_failed": "Det har skjedd en teknisk feil som gjør at du ikke kan bygge versjoner eller publisere en app til miljøene akkurat nå. Prøv igjen senere. Hvis du fortsatt har problemer, kontakt <0 href=\"mailto:tjenesteeier@altinn.no\">Altinn servicedesk</0>.",
   "app_create_release_errors.technical_error_code": "Teknisk feilkode",
-  "app_data_modelling.fields_information": "Kun objekter kan tilordnes felter i datamodelleringsverktøyet. Tekst, heltall, desimaltall og ja/nei støtter ikke egne felter. For å løse dette, velg et element som er et objekt eller konverter gjeldende element til et objekt.",
+  "app_data_modelling.fields_information": "Du kan bare tilordne objekter til felter i datamodelleringsverktøyet, ikke tekst, heltall, desimaltall eller ja/nei. Velg et element som er et objekt eller konverter gjeldende element til et objekt.",
```

Dette blir ikke helt riktig. Det vi har prøvd å si her er at det bare er objekter som kan ha underfelter. Hva med å forenkle teksten slik?
```suggestion
  "app_data_modelling.fields_information": "Hvis du vil legge til felter på dette elementet, må du konvertere det til et objekt.",
```

---

@TomasEng Er det elementet vi må konvertere til et objekt? Da kan vi spesifisere det. Restne av innholdet i strengen, skal det være med? Om tegn osv?

---

Ja. Jeg tror ikke det er nødvendig å skrive noe mer her, nei. Da ser det bare mer komplisert ut enn det er. Dette er bare en liten infomelding som kommer opp i "felter"-fanen hvis elementet ikke er et objekt.